api_v3/lib/types/KalturaUserEntry.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ * @abstract
+ */
+abstract class KalturaUserEntry extends KalturaObject implements IRelatedFilterable
+{
+
+	/**
+	 * unique auto-generated identifier
+	 * @var int
+	 * @readonly
+	 * @filter eq,in,notin
+	 */
+	public $id;
+
+	/**
+	 * @var string
+	 * @insertonly
+	 * @filter eq,in,notin
+	 */
+	public $entryId;
+
+	/**
+	 * @var int
+	 * @insertonly
+	 * @filter eq,in,notin
+	 */
+	public $userId;
+
+	/**
+	 * @var int
+	 * @readonly
+	 */
+	public $partnerId;
+
+	/**
+	 * @var KalturaUserEntryType
+	 * @readonly
+	 * @filter eq,in,notin
+	 */
+	public $type;
```

Comment:
I see no reason to expose the type in the API, the type already reflected as the object class.